### PR TITLE
Changed deviceSubId type

### PR DESCRIPTION
**Pull request by HimajaDhanyamraju2**

### CODE REVIEW

**Review by ase-101 — APPROVED** [review]

File: MockMDS/src/main/java/io/mosip/registration/mdm/dto/DeviceRequest.java [hunk 1]
```diff
@@ -17,6 +17,6 @@ public class DeviceRequest {
 	public String deviceStatus;
 	public String firmware;
 	public String certification;
-	public int[] deviceSubId;
+	public String[] deviceSubId;
```
Comment (gsasikumar) [inline MockMDS/src/main/java/io/mosip/registration/mdm/dto/DeviceRequest.java:20]: is this inline with spec?  In the spec do we have it as array of string

**Review by gsasikumar — APPROVED** [review]

**Comment by gsasikumar** [comment]
Can we check the spec before merging the changes? will this not fail our regclient and other services?

**Review by gsasikumar — CHANGES_REQUESTED** [review]
Please let me know if this is a breaking change as we are moving away from int to string? What does our spec say? int or string?

File: MockMDS/src/main/java/io/mosip/registration/mdm/dto/DeviceRequest.java (hunk 1 above)
Comment (HimajaDhanyamraju2) [reply MockMDS/src/main/java/io/mosip/registration/mdm/dto/DeviceRequest.java:20]: Yes Sasi, it is as per spec.

**Pull request merged**